src/js/base/config.js

```diff
@@ -46,3 +46,60 @@ ripe.Ripe.prototype.hasStrategy = function(strategy) {
     const strategies = (this.loadedConfig && this.loadedConfig.strategies) || ["prc"];
     return strategies.includes(strategy);
 };
+
+/**
+ * Retrieves the initials config with the specified profiles in the config. If the
+ * `profiles` argument is provided those profiles will be also used to compute the
+ * initials config.
+ *

```

Comment:
Please add a comment mentioning that this logic should be in sync with composition engine's logic when computing the initials config - currently `initials_config`